WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/CouponLineDetails.swift

```diff
@@ -120,7 +120,7 @@ struct CouponLineDetails_Previews: PreviewProvider {
         let viewModel = CouponLineDetailsViewModel(isExistingCouponLine: true,
                                                    code: "",
                                                    siteID: 0,
-                                                   didSelectSave: { _ in })
+                                                   didSelectSave: { _  in })
```

Comment:
```suggestion
                                                   didSelectSave: { _ in })
```

Reply:
Done in caa7492e597f40fbe39770b4dfe1222dc91ebef3